routes/course.js

```diff
@@ -21,5 +21,6 @@ router.param('id', idValidation)
 router.use(restrictTo(TUTOR))
 router.post('/', isEntityValid({ body }), asyncWrapper(courseController.createCourse))
 router.patch('/:id', isEntityValid({ params }),asyncWrapper(courseController.updateCourse))
+router.get('/:id', isEntityValid({ params }), asyncWrapper(courseController.getCourseById))
```

Comment:
I assume you should move the route up above the router.use(restrictTo(TUTOR)). Based on your code only tutor has an opportunity to getCourseById

Reply:
> I assume you should move the route up above the router.use(restrictTo(TUTOR)). Based on your code only tutor has an opportunity to getCourseById

you are right, fixed it